nuxeo-drive-client/nxdrive/commandline.py

```diff
@@ -37,6 +37,7 @@
 - unbind-root
 - clean_folder
 - metadata
+- share_link
```

Comment:
Can you rename to `share-link`? Arguments with underscore are standard (yes, I saw `clean_folder`, but it is a mistake).